src/app/authentication/forgot-password.component.html

```diff
@@ -7,15 +7,15 @@
 
     <form [formGroup]="resetForm" (ngSubmit)="onSubmit(resetForm.value)">
         <div class="form-group input-group" [ngClass]="{'has-error':!resetForm.controls['email'].valid && resetForm.controls['email'].dirty}">
-            <span class="input-group-addon"><i class="glyphicon glyphicon-user"></i></span>              
+            <span class="input-group-addon"><i class="glyphicon glyphicon-user"></i></span>
             <input type="email" class="form-control" name="email" placeholder="Email" [formControl]="resetForm.controls['email']">
-        </div>      
+        </div>
         <div class="form-control-feedback">
             <p *ngIf="resetForm.controls['email'].hasError('required') && (resetForm.controls['email'].dirty)">Email is required</p>
             <p *ngIf="resetForm.controls['email'].hasError('pattern') && (resetForm.controls['email'].dirty)">The email address is invalid</p>
```

Comment:
This was created by Rajat, I think he has separated them in one of his pending PRs.